spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -101,6 +101,22 @@ def __init__(self, config, spot: Spot, use_policies=True):
         self.place_env = SpotPlaceEnv(config, spot)
         self.place_env.power_robot()
 
+    def reset_env_and_policy(self, place_target, is_local):
+        """
+        Resets the place_env and policy
+
+        Args:
+            place_target (np.array([x,y,z])): Place target in either global frame or base frame of the robot
+            is_local (bool): Whether the place target is in the base frame of the robot
+
+        Returns:
+            observations: Initial observations from the place_env
+        """
+        observations = self.place_env.reset(place_target, is_local)
+        self.policy.reset()
```

Comment:
does this work if the place controller is BD API?
Maybe want to do something like:
`if self.policy is not None:
    self.policy.reset() `

Reply:
In the pipeline where `use_policies` is false, we do not call this method `reset_policy_and_environment()` . So we should be good.